content/news/speaker-support.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Speaker Support for PyLadiesCon 2024"
+meta_title: "Speaker Support for PyLadiesCon 2024"
+description: "We have speaker support for those of you who need a bit more reassurance and encouragement!
+"
+date: 2024-08-28T05:00:00Z
+categories: ["Blog Post",]
+author: "Cheuk"
+tags: ["python", "community", "conference", "public speaking", "pyladies"]
+draft: false
+layout: news
+---
+
+Since there are only 2 weeks until the [Call for Propocal](https://pretalx.com/pyladiescon-2024/cfp) closes, we do not want to you miss this speaking opportunity. Do you still need some encouragement in submitting a proposal? Maybe you need a bit more help? Don't worry! We have speaker support for those of you who need a bit more reassurance and encouragement!
+
+## Support for new speakers
+
+If you are new to public speaking and would love to get a friendly mentor to help you prepare your submission or your talk when you are accepted. Please contact us at [pyladiescon@pyladies.com](mailto:pyladiescon@pyladies.com) with the title **Looking for speaker mentor** We will try our best to find a mentor for you. All you need to do is tell us a bit about yourself (where you are based, what language you would like to speak with your mentor and what do you need help with) so that we know how we can meet your needs.
+
+## Offering your help
+
+Suppose you are an experienced speaker and would like to help mentor someone. In that case, you can also [email us](mailto:pyladiescon@pyladies.com) with the title **I want to be a speaker mentor** - tell us a bit about yourself (where you are based, what language you speak, and briefly describe your past speaking experience) so we can match you up with someone.
+
+## Technical support
+
+Since all of the speakers at PyLadiesCon 2024 are required to record their talk, we understand some of you may worry that your setup at home is not good enough for the recording or worry that you don't know how to record. We got your back! Check out our [speaker's guide here](). If you still need some help in knowing what to do, do not hesitate to [get in touch with us](mailto:pyladiescon@pyladies.com). If you think you need support in getting recording equipment, we have recording equipment grants available. Please also [contact us](mailto:pyladiescon@pyladies.com) for more information.
```

Comment:
```suggestion
Since all of the speakers at PyLadiesCon 2024 are required to record their talk, we understand some of you may worry that your setup at home is not good enough for the recording or worry that you don't know how to record. We got your back! Check out our [speaker's guide here](). If you still need some help in knowing what to do, do not hesitate to [get in touch with us](mailto:pyladiescon@pyladies.com). We have grants available to help cover the cost for your video recording equipment. If you need support, please [contact us](mailto:pyladiescon@pyladies.com) for more information.
```

Reply:
To unblock this post, perhaps we can say "we will be publishing a guide with helpful information on how to record and edit your video, so check back soon". That way we can still announce that equipment grant is available, without waiting for the full video recording guide.

Reply:
My thinking was if they are not sure they can see if they have will be good enough. Also, I think it will be good for people to know what to expect before submitting, some folks need to know that before they submit. I think the speaker guide is almost ready.